_people/NickJuty.md

```diff
@@ -4,8 +4,8 @@ id: NickJuty
 first-name: Nick
 last-name: Juty
 collection: people
-affiliation:
+affiliation: University of Manchester
```

Comment:
"**The** University of Manchester" they seem to insist.

Reply:
Is that so that they can appear Oxford and Cambridge in alphabetical lists ;) ?

Change has been made.